source/dotnet/Services/WebApi.UnitTests/Infrastructure/Integration/CalculationResultReadyIntegrationEventFactoryTests.cs

```diff
@@ -202,4 +202,93 @@ public void CreateCalculationResultCompletedForEnergySupplier_WhenQuantityQualit
             processCompletedEventDto,
             "AGlnNumber"));
     }
+
+    [Fact]
+    public void CreateCalculationResultCompletedForBalanceResponsibleParty_WhenCreating_ResultIsForBalanceResponsibleParty()
```

Comment:
nit: I think this would be better aligned with what we agreed to:
```suggestion
    public void CreateCalculationResultCompletedForBalanceResponsibleParty_ReturnsResultForBalanceResponsibleParty()
```